sdk/tests/conftest.py

```diff
@@ -119,6 +120,17 @@ def standard_router_ensembler_config_with_route_name_path():
         experiment_mappings=None,
         route_name_path="route_name",
         fallback_response_route_id="route-1",
+        lazy_routing=False,
```

Comment:
`EnsemblerStandardConfig` should be initialised with `lazy_routing`. It's not necessary to set this parameter when using `StandardRouterEnsemblerConfig` which will be initialised by the users (default value for the parameter is `False`).